programs/clearing_house/src/math/repeg.rs

```diff
@@ -428,6 +437,230 @@ pub fn adjust_peg_cost(
+            //         .ok_or_else(math_error!())?
+            //         .checked_div(terminal_base)
+            //         .ok_or_else(math_error!())?
+            //         .checked_mul(market.amm.net_base_asset_amount.unsigned_abs())
+            //         .ok_or_else(math_error!())?
+            //         .checked_sub(
+            //             new_quote_asset_reserve
+            //                 .checked_mul(market.amm.peg_multiplier)
+            //                 .ok_or_else(math_error!())?
+            //                 .checked_div(terminal_base)
+            //                 .ok_or_else(math_error!())?
+            //                 .checked_mul(market.amm.net_base_asset_amount.unsigned_abs())
+            //                 .ok_or_else(math_error!())?,
+            //         )
+            //         .ok_or_else(math_error!())?,
+            // )?;
+
+            // let adjustment_cost =
+            let _adjustment_cost =
+                amm::adjust_k_cost_and_update(&mut market_clone, &update_k_result)?;
+            per_peg_cost = cast_to_i128(market_clone.amm.quote_asset_reserve)?
+                .checked_sub(cast_to_i128(market_clone.amm.terminal_quote_asset_reserve)?)
+                .ok_or_else(math_error!())?
+                .checked_div(AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO_I128)
+                .ok_or_else(math_error!())?;
+            _adjustment_cost
+            // amm::update_k(&mut market_clone, &update_k_result)?;
+            // adjustment_cost
+        } else {
+            0
+        };
+        // let adjustment_cost: i128 = 0;
+        budget_delta_peg = budget_i128
+            .checked_add(adjustment_cost.abs())
+            .ok_or_else(math_error!())?
+            // .checked_mul(AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO_I128)
+            // .ok_or_else(math_error!())?
+            .checked_div(per_peg_cost)
+            .ok_or_else(math_error!())?;
+
+        // msg!(
+        //     "per_peg_cost: {:?}, budget: {:?}, budget_delta_peg: {:?}",
+        //     per_peg_cost,
+        //     budget_I128,
+        //     budget_delta_peg
+        // );
+
+        budget_delta_peg_magnitude = budget_delta_peg.unsigned_abs();
+        new_peg = if budget_delta_peg > 0 {
+            market
+                .amm
+                .peg_multiplier
+                .checked_add(budget_delta_peg_magnitude)
+                .ok_or_else(math_error!())?
+        } else {
+            market
+                .amm
+                .peg_multiplier
+                .checked_sub(budget_delta_peg_magnitude)
+                .ok_or_else(math_error!())?
+        };
+        cost = budget_delta_peg
+            .checked_mul(per_peg_cost)
+            .ok_or_else(math_error!())?;
+        // .checked_div(AMM_TIMES_PEG_TO_QUOTE_PRECISION_RATIO_I128)
+        // .ok_or_else(math_error!())?;
+    }
+
+    market_clone.amm.peg_multiplier = new_peg;
+
+    Ok((market_clone, cost))
+}
+
+pub fn adjust_peg_cost_cheap(
```

Comment:
looks like this is only used by `calculate_budgeted_peg` which also isnt used. if you want to keep it for future reference, it's better to put it on a branch and delete it here before merging into master